### PR TITLE
Taiwan tests lvl. 1

### PR DESCRIPTION
**Pull request by martinbenes1996**
# What type of PR is this?

Select all that apply: [ ] -> [x]

- [x] R 
- [ ] other

## R

I confirm: 

- [x] I followed the [tutorial](https://github.com/covid19datahub/COVID19/wiki/Add-a-new-data-source)
- [x] I checked the format with the [`ds_check_format`](https://covid19datahub.io/reference/ds_check_format.html) function
- [x] I double checked the data (e.g. coherent with google search)

## Additional comments

Adding to yesterday Taiwan implementation: The same webpage also has level 1 test counts.

### CODE REVIEW

File: R/ds_gov_tw.R [hunk 1]
```diff
@@ -13,16 +13,32 @@ gov_tw <- function(level, cache) {
...
+      dplyr::filter(!is.na(tests))
     
     x <- x %>% 
       dplyr::group_by_at("date") %>%
       dplyr::summarise(confirmed = sum(confirmed)) %>%
       dplyr::arrange_at("date") %>%
-      dplyr::mutate(confirmed = cumsum(confirmed))
+      dplyr::mutate(confirmed = cumsum(confirmed)) %>%
+      # add tests
+      dplyr::full_join(x.tests, by = "date")
```
Comment (jonekeat) [inline R/ds_gov_tw.R:38]: Please arrange by `date` after full join, add `%>% dplyr::arrange(date)`

**Review by jonekeat — CHANGES_REQUESTED** [review]

**Comment by jonekeat** [comment]
Hi @martinbenes1996 ,

Thanks for adding tests data in! Initially I am not sure about whether this is the actual no of test performed (since it said no of suspected cases reported through 3 channels), but I have saw [OurWorldInData](https://ourworldindata.org/coronavirus-testing#taiwan) has confirmed this

File: R/ds_gov_tw.R (hunk 1 above)
Comment (eguidotti) [reply R/ds_gov_tw.R:38]: Thanks @jonekeat for your comment. Sorting the dates before returning is not needed. This is done automatically at a top level of the package. Ready to merge!

**Comment by eguidotti** [comment]
Thanks @martinbenes1996 and @jonekeat for your help. Ready to merge!

**Pull request merged**